draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -815,6 +817,11 @@ As per {{RFC9325, Section 3.3}}, compression should not be used due to the possi
 Since some attributes of the RADIUS packets within the TLS tunnel contain values that an attacker could at least partially choose (i.e. username, MAC address or EAP message), there is a possibility for compression-related attacks, that could potentially reveal data in other RADIUS attributes through length of the TLS record.
 To circumvent this attack, this specification forbids the usage of TLS compression.
 
+The specification in {{RFC6614}} did not include any text around Server Name Indication (SNI).
+This is most likely due to the fact that the initial incentive of the specification was to just provide an easy-to-migrate-to security layer for existing RADIUS deployments.
+With the TLS layer, several features of TLS can be used to add deployment scenarios that otherwise would not have been possible.
+By adding Server Name Indication, a RAIDUS/(D)TLS server can now provide different services (i.e. access to different roaming consortia) under the same destination IP address and port, or could distinguish clients by the server name they use in the SNI extension of the TLS Client Hello and present a different server certificate or apply specific policies.
```

Comment:
I'm not clear how a server can "distinguish clients" by SNI.  Perhaps instead just "present different identities to clients based on SNI".

Reply:
Well, the simplest is a load balancer that dispatches based upon SNI.
The next simplest is a server that is basically setup a la ApacheVirtualHosts.

Reply:
How about this:

```suggestion
By adding Server Name Indication, a RAIDUS/(D)TLS server can now provide different services (i.e. access to different roaming consortia) under the same destination IP address and port, or could present different certificates to clients or apply specific policies depending the server name the clients use in the SNI extension of the TLS Client Hello.
```